diff --git a/colors/ayu.vim b/colors/ayu.vim
index 9158e6b..787cac9 100644
--- a/colors/ayu.vim
+++ b/colors/ayu.vim
@@ -33,7 +33,7 @@ let s:palette.panel     = {'dark': "#14191F",  'light': "#FFFFFF",  'mirage': "#
 let s:palette.guide     = {'dark': "#2D3640",  'light': "#D9D8D7",  'mirage': "#3D4751"}
 let s:palette.line      = {'dark': "#151A1E",  'light': "#F3F3F3",  'mirage': "#242B38"}
 let s:palette.selection = {'dark': "#253340",  'light': "#F0EEE4",  'mirage': "#343F4C"}
-let s:palette.fg        = {'dark': "#E6E1CF",  'light': "#5C6773",  'mirage': "#D9D7CE"}
+let s:palette.fg        = {'dark': "#70FF70",  'light': "#5C6773",  'mirage': "#D9D7CE"}
 let s:palette.fg_idle   = {'dark': "#3E4B59",  'light': "#828C99",  'mirage': "#607080"}
 
 "}}}
